### ISSUE TITLE
What to do with hb-shape-fuzzer on msan bot?

### ISSUE THREAD

**Issue by ebraminio** [harfbuzz/harfbuzz#1201]
As https://circleci.com/gh/harfbuzz/harfbuzz/35487 msan fails `hb-shape-fuzzer` and apparently it needs a instrumented lib*c++ be compiled there and here is how https://github.com/google/sanitizers/wiki/MemorySanitizerLibcxxHowTo#instrumented-libc

I am disabling the test for msan bot as a temporary workaround.

```
hb_shape_fuzzer: ./hb-shape-fuzzer
Uninitialized bytes in __interceptor_fopen64 at offset 86 inside [0x706000000000, 87)
==88215==WARNING: MemorySanitizer: use-of-uninitialized-value
    #0 0x7f50e10abdcf  (/usr/lib/x86_64-linux-gnu/libstdc++.so.6+0xb3dcf)
    #1 0x7f50e10ec319  (/usr/lib/x86_64-linux-gnu/libstdc++.so.6+0xf4319)
    #2 0x7f50e10ece3d  (/usr/lib/x86_64-linux-gnu/libstdc++.so.6+0xf4e3d)
    #3 0x324576  (/root/project/test/fuzzing/hb-shape-fuzzer+0x324576)
    #4 0x324758  (/root/project/test/fuzzing/hb-shape-fuzzer+0x324758)
    #5 0x7f50e0c5809a  (/lib/x86_64-linux-gnu/libc.so.6+0x2409a)
    #6 0x2ab029  (/root/project/test/fuzzing/hb-shape-fuzzer+0x2ab029)
```

**Comment by ebraminio** [comment]
Skipped, not much point on having that test on that specific bot as far as I can see.

**Comment by ebraminio** [comment]
Fixed by #1205
